card/lib/card/simplecov_helper.rb

```diff
@@ -15,7 +18,8 @@ def card_simplecov_filters
       src_file.filename =~ /mod\/#{mod}\// or 
         (
           src_file.filename =~ /tmp\// and
-          /\d+-(.+\.rb)/.match(src_file.filename) { |m| Dir["mod/#{mod}/**/#{m[1].gsub("-","/")}"].present? } 
+          /\d+-(.+\.rb)/.match(src_file.filename) { |m| Dir["mod/#{mod}/**/#{m[1].gsub("-","/")}"].present? } and
+          is_in_mod? src_file,mod
```

Comment:
should probably separated this matching out into a separate method as well (and give it a name so that we know what it's doing)